File: app/src/main/java/com/example/dcct/model/CoverModel.java
package com.example.dcct.model;

import com.example.dcct.bean.CoverEntity;

import java.util.List;

public interface CoverModel {
    void requestImageUrl(OnReturnListener onReturnListener);
    interface OnReturnListener{
        void onComplete(List<CoverEntity> coverEntities);
        void onError(String msg);
    }
}

File: app/src/main/java/com/example/dcct/model/CoverModel.kt
package com.example.dcct.model

import com.example.dcct.bean.BackResultData
import com.example.dcct.bean.CoverEntity

interface CoverModel {
    fun requestImageUrl(onReturnListener: OnReturnListener)
    interface OnReturnListener {
        fun onComplete(coverEntities: List<CoverEntity>)
        fun onError(msg: String)
    }
}